data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,462 @@
+	landing
+	source "Hai-home"
+	stopover "Makerplace"
+	stopover "Darkmetal"
+	stopover "Giverstone"
+	stopover "Stonebreak"
+	stopover "Cloudfire"
+	stopover "Snowfeather"
+	stopover "Icelake"
+	stopover "Newhome"
+	stopover "Frostmark"
+	stopover "Heartvalley"
+	stopover "Allhome"
+	stopover "Dustmaker"
+	stopover "Greenbloom"
+	stopover "Skyfarm"
+	stopover "Farwater"
+	stopover "Greenwater"
+	destination "Mountaintop"
+	clearance
+	passengers 1
+	blocked `Terry contacts you and says, "Captain, you're going to need a bunk free for me. Do you have space?" You do not, so you should complete or abort another mission to make room for her, then return here.`
+	on visit
+		dialog `You have arrived at the Mountaintop construction site, but it isn't finished or staffed yet.`
+	to offer
+		has "Hai Building Mountaintop [1b] Return: done"
+	on offer
+		conversation
+			`At Hai-home, you are directed to a now familiar government hangar. Sayari, Xilin, and Alondo all send you greeting pings, but are busy at the time of your landing. To your surprise though, Terry is banging on your airlock before too long.`
+			`	"Captain <last>, or should I call you <first>? Anyway, doesn't matter. How would you like to make some money?" The smile she gives you is something like you imagine a large cat giving the tasty morsel it just identified as its food. "Of course you do, everyone likes to make money," she continues, not waiting for you to reply.`
+			`	"The Hai have put out the call for volunteers to staff the new Mountaintop station when it's finished. While you've been off watching early construction works, we've been gauging public interest and well... it's not going to work. In order to sell this, they're going to need at least four thousand volunteers by the end of the week to get a good handle on what they can expect to need in terms of facilities, amenities, supplies, and various livability requirements. Nearly twelve thousand by the time the station is actually built. Our current estimates suggest they'll only get a quarter of that unless something changes."`
+			`	As she's talking, Terry has been laying out charts and graphs while using a handheld Hai holographic device to show animated diagrams.`
+			choice
+				`	"How do you know all this?"`
+			`	Terry raises an eyebrow at you, "You remember we have a supercomputer that could predict the economic situation across the whole of human habited space up to fifty years in the future to within 1% accuracy right?"`
+			`	When she puts it that way, it does suddenly feel like a bit of a stupid question.`
+			# And how did that turn out for you eh? lol
+			`	"As it turns out, that same computer is predicting that the Syndicate stands to lose billions of credits, per day, for the duration of any crisis evolving from this which involves the uncontrolled release of the Hai secret; however, we have a plan."`
```

Comment:
Yeah I know there's a push to destroy or otherwise re-purpose the Oracle both by you and some other people. (though as you note I don't know the specific ideas in your head.)

I would really like to see the Oracle explored a little differently though. It's weird that the Oracle is the only AI of it's type in human space, and I'd like to see that explored. My thinking is that any sort of _networked_ AI ultimately goes insane, so they're not used. Given the obvious value of it though, I think it would be fun if (rather than it being networked to be able to draw down all the info it wanted) it was physically isolated to remain sane, and had to be actively fed all the information that made it tick. 
This would then mean that there was a massive infrastructure of corporate data collectors, compilers, and reporters, all feeding back huge volumes of information from across human space to be plugged into the Oracle's databanks so it can do its magic.
You could have a whole department of the Syndicate with a secret purpose, a whole information industry dedicated to making sure that the right information was collected, reported on, and sent to the Oracle for use. I think that has a lot of scope for interesting narrative outcomes - and then the Syndicate could keep their Oracle, with some oversight, because it would be functionally useless without the information industry that feeds it.